packages/eslint-config-spt-react/react.js

```diff
@@ -18,14 +18,14 @@ module.exports = {
         'react/jsx-sort-prop-types': 'off',
         'react/jsx-uses-react': 'error',
         'react/jsx-uses-vars': 'error',
-        'react/no-did-mount-set-state': ['error', 'allow-in-func'],
+        'react/jsx-wrap-multilines': 'error',
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1404810/17967074/90a83452-6ac5-11e6-9ee6-bd00677466ee.png)

Also moved to keep things alphabetical